.travis.yml

```diff
@@ -44,9 +44,6 @@ matrix:
     - rvm: rbx
       env: RUN=rspec
 
-    - rvm: jruby
-      env: RUN=rspec
-
```

Comment:
this is no longer necessary: 23d73a544d (Use the latest jRuby stable version on Travis) :)